src/drivers/voxl2_io/voxl2_io.cpp

```diff
@@ -575,6 +572,22 @@ void Voxl2IO::Run()
 
 	perf_begin(_cycle_perf);
 
+	/* Verify protocol version info */
+	if (_need_version_info){
+		if (get_version_info() < 0) PX4_ERR("Failed to detect voxl2_io protocol version.");
+		if (_version_info.sw_version == VOXL2_IO_SW_PROTOCOL_VERSION && _version_info.hw_version == VOXL2_IO_HW_PROTOCOL_VERSION){
+			_need_version_info = false;
+			PX4_INFO("Detected M0065 protocol version. SW: %u HW: %u", _version_info.sw_version, _version_info.hw_version);
+		} else if (_protocol_read_retries > 0) {	// If Voxl2 IO gets powered up late, the initial values read are sometimes garbage
+			PX4_ERR("Detected incorrect M0065 protocol version. SW: %u HW: %u. Retrying, %i attempts left...", _version_info.sw_version, _version_info.hw_version, _protocol_read_retries--);
```

Comment:
Is this really an error?

Reply:
This has been updated in commit [d69b600](https://github.com/modalai/px4-firmware/pull/50/commits/d69b60038b27befb45d73bd44bf01b56d99f2bc8).